ampd/src/broadcaster_v2/mod.rs

```diff
@@ -84,6 +99,64 @@ where
     key_id: String,
     gas_adjustment: f64,
     gas_price: DecCoin,
+    #[builder(default = None, setter(strip_option))]
+    tx_confirmer_client: Option<confirmer::TxConfirmerClient>,
+}
+
+#[allow(dead_code, non_camel_case_types)]
```

Comment:
is this not used?

Reply:
I probably forgot to remove it when copying the old builder from the expanded macro